tango/server.py

```diff
@@ -613,6 +616,11 @@ def get_device_properties(self, ds_class=None):
                 value = self.prop_util.get_property_values(
                     prop_name, self.device_property_list)
                 self._tango_properties[prop_name] = value
+                properties = self.device_property_list[prop_name]
+                mandatory = properties[3]
+                if mandatory is True and value is None:
```

Comment:
`mandatory` might be any truthy value, so maybe `if mandatory and value is None`?